apps/nextjs-website/src/lib/strapi/codecs/NullToUndefined.ts

```diff
@@ -0,0 +1,27 @@
+import * as t from 'io-ts/lib';
+import { pipe } from 'fp-ts/lib/function';
+import * as E from 'fp-ts/lib/Either';
+
+/**
+ * This type is used to convert null to undefined when decoding.
+ * This is useful becuase Strapi returns null for missing fields, so we want to convert those to undefined to be more consistent with the rest of our codebase.
+ *
+ * @example
+ * const codec = t.union([NullToUndefined, t.string]);
+ * const result = codec.decode(null);
+ * // result will be Right(undefined)
+ */
+export const NullToUndefined = new t.Type<undefined, null, unknown>(
```

Comment:
Maybe we should rename this constant (and the file) to `NullToUndefinedCodec` to be consistent with `BlocksContentCodec`. @MarcoPonchia @tommaso1 @datalek @kin0992 What do you think about this?

Reply:
Given that for me it's fine both with and without Codec. Having said that, I see the Codec suffix a good fit for complex types (e.g. structures). I see `NullToUndefined` more like [DateFromISOString](https://github.com/gcanti/io-ts-types/blob/b3a5616/src/DateFromISOString.ts#L24)